backend/src/controllers/onboardingController.ts

```
@@ -0,0 +1,512 @@
+      if (data.emergencyContacts?.length > 0) {
+        await prisma.emergencyContact.createMany({
+          data: data.emergencyContacts.map((contact: any) => ({
+            ...contact,
+            patientId,
+          })),
+        });
+      }
+      break;
+
+    case 3: // Medical History
+      const { allergies, medications, conditions, ...otherHistory } = data.medicalHistory;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)